episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,498 @@
+'mitocheck.ome.tiff' dataset. You should have already downloaded this as part of 
+the [setup instructions](../learners/setup.md).
+
+Run the code below in Napari's console. This will save a specific timepoint of 
+this image (time = 30) as four different file formats (.tiff, .png + low and 
+high quality .jpg). Note: these files will be written to the same folder as the 
+'mitocheck.ome.tiff' image!
+
+```python
+from skimage.io import imsave
+from pathlib import Path
+
+# Get the mitocheck.ome layer, and get timepoint = 30
+layer = viewer.layers["mitocheck.ome"]
+image = layer.data[30, :, :]
+
+# Save as different file formats in same folder as mitocheck.ome
+folder_path = Path(layer.source.path).parent
+imsave( folder_path / "test-tiff.tiff", image)
+imsave( folder_path / "test-png.png", image)
+imsave( folder_path / "test-jpg-high-quality.jpg", image, quality=75)
+imsave( folder_path / "test-jpg-low-quality.jpg", image, quality=30)
+
+```
+
+- Go to the folder where 'mitocheck.ome.tiff' was saved and look at the file 
+sizes of the newly written images (`test-tiff`, `test-png`, 
+`test-jpg-high-quality` and `test-jpg-low-quality`). Which is biggest? 
+Which is smallest?
+
+- Open all four images in Napari. Zoom in very close to a bright nucleus, and 
+try showing / hiding different layers with the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility.svg
+){alt="A screenshot of Napari's eye button" height='30px'} icon. 
+How do they differ? How does each compare to timepoint 30 of the original 
+'mitocheck.ome' image?
+
+- Which file formats use lossy compression?
+
+:::::::::::::::::::::::: solution 
+ 
+## Solution
+
+### File sizes
+
+'test_tiff' is largest, followed by 'test-png', then 'test-jpg-high-quality', 
+then 'test-jpg-low-quality'.
+
+### Differences to original
+
+By showing/hiding different layers, you should see that 'test-tiff' and 
+'test-png' look identical to the original image (when its slider is set to 
+timepoint 30). In contrast, both jpg files show blocky artefacts around the 
+nuclei - the pixel values have clearly been altered. This effect is worse in the 
+low quality jpeg than the high quality one.
+
+### Lossy compression
+
+Both jpg files use lossy compression, as they alter the original pixel values. 
+The low quality jpeg compresses the file more than the high quality jpeg, 
+resulting in smaller file sizes, but also worse alteration of the pixel values. 
+The rest of the file formats (.tiff / .png) use no compression, or lossless 
+compression - so their pixel values are identical to the original values. 
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+### Handling of large image data
+
+If your images are very large, you may need to use a pyramidal file format that 
+is specialised for handling them. Pyramidal file formats store images at 
+multiple resolutions (and usually in small chunks) so that they can be browsed 
```

Comment:
Dan provided a nice analogy to Google maps that I think is intuitive here for people to understand. 
Are there any handy pictures around that we can use to show/describe this?